outline.md

```diff
@@ -0,0 +1,44 @@
+# Learning Outline 
+
+## Content
+
+The workshop is divided into the following three sections:
+
+### The Unix Shell
+
+The aim of this section is not to teach learners basic unix commands,
+or syntax for bash scripting,
+but rather to motivate them
+to **automate tasks and develop pipelines**.
+Topics covered will include filesystem navigation and manipulation,
+redirection and piping of standard input and output,
+file permissions, and shell scripts.
+
```

Comment:
We will also need to cover using `ssh` since that is the typical method to interact with remote resources..